src/utils/mavenUtils.ts

```diff
@@ -16,6 +20,16 @@ export namespace mavenUtils {
         try {
             await cpUtils.executeCommand(undefined, workingDirectory, mvnCommand, '--version');
         } catch (error) {
+            const settingKey: string = 'showMavenPathWarning';
+            if (getFuncExtensionSetting<boolean>(settingKey)) {
+                const message: string = localize('azFunc.mvnNotFound', 'Failed to find "maven", please ensure that the maven bin directory is in your system path.');
+                const result: vscode.MessageItem = await ext.ui.showWarningMessage(message, DialogResponses.learnMore, DialogResponses.dontWarnAgain);
+                if (result === DialogResponses.learnMore) {
+                    await opn('https://aka.ms/azurefunction_maven');
```

Comment:
Don't forget to update the short url after PR merged. Currently it is pointing to your own repo.